docs/markdown/getting-started.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started
+
+This tutorial will help you start an Interledger node locally on you computer and create a local ILP test network.
+
+This tutorial describes how to:
+1. Install [`moneyd`](../concepts/moneyd.md) on your system
+2. Start an [Interledger](https://github.com/interledgerjs/ilp-connector) node on a local test network using moneyd
+3. Send and receive value using the [SPSP](../concepts/spsp.md) (Simple Payment Setup Protocol) API
+
+![Getting-Started-Locally](../images/getting-started-locally.png)
+
+## Before you begin
+* Install a stable version of [Node.js](https://nodejs.org/en/) (10.16.0 LTS is recommended)
+* Set up Node to install modules globally <br/>
```

Comment:
Doesn't this happen automatically? If not, you should link to some description of how to do this

Reply:
https://docs.npmjs.com/files/folders

Looks like it does. I'll remove it.